tests/test_base.py

```diff
@@ -636,3 +655,189 @@ def test_pack_header_for_user(self, app, user_class):
             assert token_data['rls'] == 'admin,operator'
             assert token_data['duder'] == 'brief'
             assert token_data['el_duderino'] == 'not brief'
+
+    def test_registration_email(self, app, user_class, db):
+        """
+        This test verifies that Praetorian encrypts passwords using the scheme
```

Comment:
wrong docstring for this test?

Reply:
yeah, i yy/p my code a lot .. told you i'm lazy with docs

Reply:
submitted